find-successful-workflow.ts

```diff
@@ -88,6 +91,23 @@ function reportFailure(branchName) {
     - If no, then you might have changed your git history and those commits no longer exist.`);
 }
 
+function createOctokitClient() {
+  const OctokitClient = Octokit.plugin(
```

Comment:
Let's just have this as a const e.g. `ProxifiedClient` and call constructor where needed.

There is another instance of `octokit` being created at `findMergeQueueBranch` function.